docs/src/testing/testing.md

```diff
@@ -143,3 +143,21 @@ Tests: 1 passed, 0 failed, 0 skipped, 0 ignored, 0 filtered out
 ```
 
 For more information about how starknet-foundry calculates those, see [gas and resource estimation](gas-and-resource-estimation.md) section.
+
+
+## Using External Contracts In Tests
```

Comment:
I would rather have it as a separate section in the contract-collection.md file I mentioned about above

Reply:
I would even resort to a shorter description than this, and linking to the scarb docs since their description is pretty good. We can leave the example for ease of copy-pasting